internal/trigger/messagebus/messaging.go

```diff
@@ -57,7 +58,7 @@ func (trigger *Trigger) Initialize(logger logger.LoggingClient) error {
 			case msgErr := <-messageErrors:
 				logger.Error(fmt.Sprintf("Failed to receive ZMQ Message, %v", msgErr))
 			case msgs := <-trigger.topics[0].Messages:
-				logger.Debug("Received message from bus")
+				logger.Trace(fmt.Sprintf("Received message from bus on topic '%s'. Corrilation Id is %s", trigger.Configuration.Binding.PublishTopic, msgs.CorrelationID))
```

Comment:
Corrilation should be "Correlation"

Reply:
Consider also possibly making the topic a key with the name as a value so the logging is semi-structured.